src/commands/removetask.js

```diff
@@ -0,0 +1,86 @@
+const { ClientRequest } = require('http');
+const { client } = require('tmi.js');
+const database = require('../database');
+const getDay = require('../util/getDay');
+const isMod = require('../util/isMod');
+
+module.exports = function(context) {
+  const { client, target } = context;
+
+  // data validation
+  if (!context?.['display-name'] ||
+      !context?.['user-id']) {
+    console.log('validation failed');
+    return;
+  }
+
+  // get user
+  const user = {
+    id: context['user-id'],
+    name: context['display-name'],
+  };
+  
+  // clear list if out of date
+  const today = getDay();
+  let todoTable = database.get('todoTable');
+  todoTable = todoTable || {
+    day: today,
+    tasks: [],
+  };
+
+  if (todoTable.day) {
+    if (todoTable.day !== today) {
+      todoTable = {
+        day: today,
+        tasks: [],
+      };
+    }
+  }
+  database.set('todoTable', todoTable);
+
+  const userIsMentioned = context.variables.length === 1 && context.variables[0].substring(0,1) === '@';
+
+  // remove all tasks if mod is removing for user (in case TOS), else remove active task for author
+  if (isMod(user.id) && userIsMentioned) {
```

Comment:
What about the case where userIsMentioned is true by isMod(user.id) is false? Should we throw an error in this case, or just go through the remove own task flow?